File: src/BukkitPE/event/player/PlayerEvent.php
<?php
/**
 * Player-only related events
 */
namespace BukkitPE\event\player;

use BukkitPE\event\Event;

abstract class PlayerEvent extends Event{
	/** @var \BukkitPE\Player */
	protected $player;

	public function getPlayer(){
		return $this->player;
	}
}

File: src/main/java/net/BukkitPE/event/player/PlayerEvent.java
package net.BukkitPE.event.player;

import net.BukkitPE.Player;
import net.BukkitPE.event.Event;

/**
 * BukkitPE Project
 */
public abstract class PlayerEvent extends Event {
    protected Player player;

    public Player getPlayer() {
        return player;
    }
}
